### PR TITLE
feat: add monte carlo page

### PR DESCRIPTION
**Pull request by woutervanranst**
@LukV can you review and merge?

### CODE REVIEW

File: docs/value-added-apis/monte_carlo.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+## Future Data
+
+The endpoint has a range of predefined fields to queery data for. Let us look at an example.
+
+
+=== "HTTP"
+
+    ```HTTP 
+    POST /performance/future/?X-TENANT-ID={your_identifier} HTTP/1.1
```
Comment (LukV) [inline docs/value-added-apis/monte_carlo.md:24]: This is not how to add a header to an HTTP request. More like:

POST /performance/future/ HTTP/1.1
Host: api.data.uat.investsuite.com
X-TENANT-ID: demo.com
Content-Type: application/json
Authorization: Bearer {access_token_string}

File: docs/value-added-apis/monte_carlo.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    Host: api.data.uat.investsuite.com
+    Content-Type: application/json
+    Authorization: Bearer {access_token_string}
```
Comment (LukV) [inline docs/value-added-apis/monte_carlo.md:27]: You do not apply a JWT correct, more like a key as query param? So Authorization: Bearer {access_token_string} is not correct here

File: docs/value-added-apis/monte_carlo.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            "end_date": "2022-12-31",
+            "currency": "EUR",
+            "start_amount": 1000,
+            "recurring_deposit_amount": 50,
+            "recurring_deposit_frequency": "M",
+            "sample_frequency": "M",
+            "quantiles": [0.05, 0.5, 0.95],
+            "use_expected_returns": false
+        }'
+    https://api.data.uat.investsuite.com/performance/future/batch/?X-TENANT-ID={your_identifier}
```
Comment (LukV) [inline docs/value-added-apis/monte_carlo.md:68]: See remark above

**Comment by woutervanranst** [comment]
@nico-ivs can you review/incorporate @LukV's comments?

File: docs/value-added-apis/monte_carlo.md (hunk 1 above)
Comment (johnraz) [reply docs/value-added-apis/monte_carlo.md:24]: @LukV is correct 👍

File: docs/value-added-apis/monte_carlo.md (hunk 2 above)
Comment (johnraz) [reply docs/value-added-apis/monte_carlo.md:27]: Indeed the authentication is done with `X-API-KEY` header.

**Pull request merged**